File: code/storage/EletricGo/Domain/Deliveries/Product.cs
using EletricGo.Domain.Shared;

namespace EletricGo.Domain.Deliveries {
    public class Product : Entity<ProductId> {
        public string Name { get; set; }
        public double Weight { get; set; }
        public float LevelOfPolution { get; set; }


        public Product() {
        }

        public Product(string name, double weight, float levelOfPolution) {
            Id = new ProductId(Guid.NewGuid());
            Name = name;
            Weight = weight;
            LevelOfPolution = levelOfPolution;
        }

    }
}

File: code/spa/src/models/product.ts
export default class Product {
    Name: string;
    Weight: number;
    LevelOfPolution: number;
    Delivery: string;

    constructor(name: string, weight: number, levelOfPolution: number, delivery: string) {
        this.Name = name;
        this.Weight = weight;
        this.LevelOfPolution = levelOfPolution;
        this.Delivery = delivery;
    }
}